go/libkb/logout.go

```diff
@@ -176,6 +177,8 @@ func CanLogout(mctx MetaContext) (res keybase1.CanLogoutRes) {
 		return res
 	}
 
+	mctx, cancel := mctx.WithTimeout(5 * time.Second)
```

Comment:
cc @patrickxb, I'm putting at imeout on this, otherwise it stalls for 20-30+ seconds polling merkle. Sound ok?

Reply:
You mean if you're offline?

Reply:
Yes

Reply:
ok merging this, lmk if you want me to back it out.